database/helpers.go

```diff
@@ -8,33 +8,38 @@ import (
 var separator = []byte("/")
 
 // Key is a helper type meant to combine prefix
-// and keys into a single full key-value
+// and suffix into a single full key-value
```

Comment:
Please don't use the phrase "key-value" as it means something completely different.